portfolio/src/main/webapp/script.js

```diff
@@ -111,6 +58,36 @@ function createListElement(text) {
   return liElement;
 }
 
+function createPostElement(postJson) {
```

Comment:
You can simplify this by just creating a string  of the HTML contents, i.e.
`post.innerHTML = "<div><div><p id="username"></p><p id="timestamp"></p></div><div><img id="image"></div><div><p id="comment"><p></div></div>"`

`document.getElementById("timestamp").innerText = postJson.timeStamp;`
...

Reply:
I considered this, but then it doesn't make my Post extendable or more readable, I was also considering that I would add more elements to the Post object like number of likes etc and wanted to document the structure of the post element.

Reply:
will change to plaintext as discussed in the meeting :)